test/multiverse/suites/async_http/Envfile

```diff
@@ -0,0 +1,19 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+ASYNC_HTTP_VERSIONS = [
+  nil,
+  '0.59.0'
+]
```

Comment:
Looks like the RubyGems site is misleading about the lowest supported version. Could you add constraints to test only Ruby 2.5 and above? 

```
Bundler could not find compatible versions for gem "ruby":
  In Gemfile.1:
    ruby

    async-http (~> 0.59.0) was resolved to 0.59.5, which depends on
      async-io (>= 1.28) was resolved to 1.30.0, which depends on
        ruby (~> 2.5)

    newrelic_rpm was resolved to 9.5.0, which depends on
      ruby (>= 2.4.0)
```

Reply:
added 2.5 as the min ruby version https://github.com/newrelic/newrelic-ruby-agent/pull/2272/commits/e911c78feb1ffc87524bc380577b133ab37820d7